doc/fileformat/cable_cell.rst

```diff
@@ -0,0 +1,429 @@
+.. _formatcablecell:
+
+Cable cell format
```

Comment:
This sounds a bit too non specific. I suppose it's one of those hard things, but I think we need one (The Arbor Cable Cell Format). Or, are we so ambitious (maybe we are!) that this in indeed _The_ Cable Cell Format?

Reply:
I kept it vague because I didn't know what to call it. I'll bring up the naming issue on slack.